lib/elements/dom-repeat.js

```diff
@@ -542,18 +549,16 @@ export class DomRepeat extends domRepeatBase {
     const isntIdxToItemsIdx = this.__sortAndFilterItems(items);
     // If we're chunking, increase the limit if there are new instances to
     // create and schedule the next chunk
-    if (this.initialCount) {
-      this.__updateLimit(isntIdxToItemsIdx.length);
-    }
+    this.__updateLimit(isntIdxToItemsIdx.length);
```

Comment:
Sorry should have made that clear.  It's because `__itemsArrayChanged` needs to be set to false unconditionally after each render, regardless of `initialCount`.